proxy/http2/Http2ConnectionState.cc

```diff
@@ -1450,14 +1450,16 @@ Http2ConnectionState::send_a_data_frame(Http2Stream *stream, size_t &payload_len
 }
 
 void
-Http2ConnectionState::send_data_frames(Http2Stream *stream)
+Http2ConnectionState::send_data_frames(Http2Stream *stream, bool &stream_deleted)
 {
+  stream_deleted = false;
```

Comment:
Is this really necessary? Isn't it set already before you call send_data_frames() ?